### ISSUE TITLE
Avoid error DL1000

### ISSUE THREAD

**Issue by Hykire** [hadolint/hadolint#461]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
Avoid the error DL1000
### Actual behavior
Shows the error
### Steps to reproduce the behavior
create a dockerfile with the code below :**

Invoke the hadolint like this:
hadolint -f json --ignore DL1000 Dockerfile
But i get this error:
Error: code":"DL1000", echo -e '\\n\\\r\n   |     ^\nunexpected 'e'\nexpecting '#', '\\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, at least one space, or end of input\n"

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.18.0-6-ga0d655d
```

**Dockerfile** (if relevant)

```Dockerfile
FROM python:3.6.11-alpine3.11
RUN set -eux; \
# hadolint ignore=DL1000
    echo -e '\n\
@edge_main http://dl-cdn.alpinelinux.org/alpine/edge/main\n\
@edge_comm http://dl-cdn.alpinelinux.org/alpine/edge/community\n\
@edge_test http://dl-cdn.alpinelinux.org/alpine/edge/testing'\
    >> /etc/apk/repositories; 
```

**Additional environment details (OS, stack version, etc.)**
hadolint/hadolint:latest

**Comment by lorenzo** [comment]
You need to put the ignore instruction on top of the `RUN` like this

```dockerfile
# hadolint ignore=DL1000
RUN ...
```
